app-creator/handlers/templates/templates.go

```diff
@@ -0,0 +1,63 @@
+package handlers
+
+import (
+	"log"
+	"net/http"
+	data "modules/data/templates"
+	"cloud.google.com/go/datastore"
+)
+
+type Templates struct {
+	l *log.Logger 
```

Comment:
A thought: you can embed the log.Logger; see example and reasoning in https://golang.org/doc/effective_go.html#embedding (`type Job struct`).

Reply:
I'm using the logger in another file data.go. If I want to embed the logger, I will have to move this logic to the templates.go file along with other values. Otherwise, I will have circular imports. Will it be better to move it over?
```
func Get(ctx context.Context, db *datastore.Client, l *log.Logger) Templates {
	...
	if err != nil {
		l.Fatal("Unable to fetch templates\n")
	}
        ...
}
```